source/includes/steps-import.yaml

```diff
@@ -19,25 +21,57 @@ content: |
 
   .. figure:: /images/compass/import-data-dialog.png
      :alt: Import data dialog
+     :width: 60%
 
 ---
-title: Choose the appropriate file type of the source data file. 
+title: Select the location of the source data file under 
+       :guilabel:`Select File`.
+level: 4
+ref: click-file-location
+content: ""
+
+---
+title: Choose the appropriate file type.
 level: 4
 ref: choose-file-type
 content: |
-  Under :guilabel:`Select Input File Type`, select either JSON or CSV. 
+
+  Under :guilabel:`Select Input File Type`, select either 
+  :guilabel:`JSON` or :guilabel:`CSV`. 
+
+  If you are importing a CSV file, you may specify fields to import and 
+  the types of those fields under :guilabel:`Specify Fields and Types`:
```

Comment:
Does Compass automatically try to guess the type? Is there a default value? Might be worth clarifying what the "default" behavior is for data types.

Reply:
Good call - defaults to strings.